crates/runtime/src/http/v1/mod.rs

```diff
@@ -262,6 +271,11 @@ fn attach_cache_headers(headers: &mut HeaderMap, results_cache_status: CacheStat
     {
         headers.insert("Results-Cache-Status", val);
     }
+
+    // Tell CDN entry is unique per user cache key
+    if user_key_specified {
+        headers.insert("Vary", HeaderValue::from_static("X-Spice-Cache-Key"));
```

Comment:
```suggestion
        headers.insert("Vary", HeaderValue::from_static("Spice-Cache-Key"));
```

https://www.rfc-editor.org/rfc/rfc6648